src/libtfhe/toruspolynomial-functions.cpp

```diff
@@ -136,3 +136,41 @@ EXPORT double intPolynomialNormSq2(const IntPolynomial* poly){
     }
     return temp1;
 }
+
```

Comment:
it may never have been defined (intpolynomial functions were not used in the main code). do you need this function?